portfolio/src/main/webapp/script.js

```diff
@@ -17,10 +17,10 @@
  */
 
  //random image generator 
```

Comment:
Sentences in comments start with a capitalized word and end with a period, and there should be a blank after "//"
e.g.:
// Generate random image.

Try to find a jslint or clang-format tool to fix this automatically.(search in moma)

Reply:
ah ok that comment is from the initial commit from before, I didn't change it

Reply:
i'll look for such a tool